internal/render/render.go

```diff
@@ -18,16 +19,41 @@ var (
 )
 
 func init() {
+	initEmptyTmpls()
+	for _, name := range bindata.AssetNames() {
+		tmpls = template.Must(tmpls.Parse(string(bindata.MustAsset(name))))
+	}
+}
+
+// LoadCustomTemplate allows to load in custom tamplates from a specified path
```

Comment:
s/tamplates/templates

Also, period at the end of the comment.

Reply:
👍